data/src/main/java/com/ionutconstantinnicolae/data/remote/HttpSuccessInteceptor.java

```diff
@@ -0,0 +1,28 @@
+package com.ionutconstantinnicolae.data.remote;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+//class HttpSuccessInterceptor implements Interceptor {
```

Comment:
cleanup?

Reply:
it seems like we do need this class in order to map our network to out business exceptions